docs/stdlib/json.rst

```diff
@@ -91,39 +93,37 @@ expression <select>` in *JSON mode*, including the type shape.
     ...     filter .name = 'std::bool');
     {'{"name": "std::bool", "timestamp": "2019-04-02"}'}
 
-JSON values can also be cast back into scalars. This casting is
-symmetrical meaning that if a scalar can be cast into JSON, only that
-particular JSON type can be cast back into that scalar:
+JSON values can also be casted back as a scalar type. Casting JSON is
+symmetrical: if a scalar type can be casted back into JSON, only that
+particular JSON type can be casted back into that scalar. Some scalar types
+will have specific conditions:
```

Comment:
This hits my ears funny. I think it's mostly because of "JSON type" since, if I understand, EdgeDB doesn't have a concept of a JSON type. The type is JSON or it's something else. JSON data itself has types, and that's what we're talking about here.

I might fall back on my safety net of "compatible" as a compromise here. Something along these lines:

```
JSON values can also be cast back into scalars. Casting JSON is symmetrical:
if a scalar value can be cast into JSON, a compatible JSON value can be cast
into a scalar of that type. Some scalar types will have specific conditions for
casting:
```